.circleci/config.yml

```diff
@@ -0,0 +1,32 @@
+version: 2.1
+orbs:
+  go: circleci/go@1.5.0
+  aws-cli: circleci/aws-cli@1.4.0
+jobs:
+  test:
+    executor:
+      name: go/default
+      tag: '1.16'
```

Comment:
Go version 1.16 is required for fs.FS

Reply:
Hmm, would that make go-packages incompatible with anything prior to 1.16? If so we might not be able to make the upgrade.

Reply:
The fs.FS is causing this. We can strip it out until we are ready

Reply:
Using some build tags we can now use builder with pre go 1.16 just without the extra fs.FS magic. updated circleci to test both versions.